corehq/apps/email/migrations/0003_emailsettings_password_cbc.py

```diff
@@ -12,7 +12,7 @@ def copy_and_reencrypt_password_to_password_cbc(apps, schema_editor):
     EmailSettings = apps.get_model('email', 'EmailSettings')
 
     for email_settings in EmailSettings.objects.all():
-        email_settings.password_cbc = reencrypt_ecb_to_cbc_mode(email_settings.password, f'${ALGO_AES}$')
+        email_settings.password = reencrypt_ecb_to_cbc_mode(email_settings.password, f'${ALGO_AES}$')
         email_settings.save()
```

Comment:
This will load all rows into memory at once, and then save each one individually, which has two potential problems:
- May use a lot of memory.
- Will take a long time because many individual updates are much less efficient than bulk updates.

What do you think of reimplementing this to load rows in batches and update each batch with a single batch update query?